src/applications/personalization/preferences/actions/index.js

```diff
@@ -25,7 +25,7 @@ export function fetchUserSelectedBenefits() {
       '/user/preferences',
       null,
       response => {
-        // We just want to get an array of Benefits preferences);
+        // We just want to get an array of Benefits preferences
         let selectedBenefits = response.data.attributes.userPreferences;
```

Comment:
We might rename this user preferences here and selected benefits below, in case that helps to clarify the shape the data should be.